packages/remix-dev/config.ts

```diff
@@ -726,11 +737,18 @@ const resolveServerBuildPath = (
 
 // @ts-expect-error available in node 12+
 // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat#browser_compatibility
-let listFormat = new Intl.ListFormat("en", {
+let conjunctionListFormat = new Intl.ListFormat("en", {
```

Comment:
What is this magic doing?

Reply:
basically just converts an array to "x and y", "x, y, and z", "x or y", "x, y, or z"